src/plugins/auto/src/plugin.cpp

```diff
@@ -278,7 +279,7 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
     } else if (METRIC_KEY(SUPPORTED_CONFIG_KEYS) == name) {
         return m_plugin_config.supported_rw_properties(get_device_name());
     OPENVINO_SUPPRESS_DEPRECATED_END
-    } else if (ov::supported_properties == name) {
+    } else if ((ov::supported_properties == name) || (ov::internal::supported_properties == name)) {
```

Comment:
Is this correct impl? for GPU you return only internal properties for `ov::internal::supported_properties` query, but here I assume that auto plugin will return all supported properties (union of internal + external), won't it?

Reply:
> Is this correct impl? for GPU you return only internal properties for `ov::internal::supported_properties` query, but here I assume that auto plugin will return all supported properties (union of internal + external), won't it?

Fixed